### PR TITLE
Cull removed rules.

### PR DESCRIPTION
**Pull request by ralphbean**
In fedora-infra/fmn.rules#21 a few rules got removed.  This new alembic
upgrade script goes through and removes them from the db.

### CODE REVIEW

File: alembic/versions/2136a1f22f1f_cull_removed_rules.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+def upgrade():
+    engine = op.get_bind().engine
+    session = sa.orm.scoped_session(sa.orm.sessionmaker(bind=engine))
+
+    # Find all rules that got removed in this PR and nuke them
+    # https://github.com/fedora-infra/fmn.rules/pull/21
+    goners = [
+        'fmn.rules:pkgdb_acl_user_remove',
+        'fmn.rules:pkgdb_branch_clone',
+        'fmn.rules:pkgdb_critpath_update',
```
Comment (pypingou) [inline alembic/versions/2136a1f22f1f_cull_removed_rules.py]: This one was moved, not removed: https://github.com/fedora-infra/fmn.rules/pull/21/files#diff-f1b769f34e51b7aa715c3fda9ab9f599R123

**Comment by pypingou** [comment]
One topic too many otherwise :+1:

Do you check if we need/have a behavior defined ONDELETE?

File: alembic/versions/2136a1f22f1f_cull_removed_rules.py (hunk 1 above)
Comment (ralphbean) [inline alembic/versions/2136a1f22f1f_cull_removed_rules.py]: > This one was moved, not removed

Fixed in 1a37b17.  Thanks!

**Comment by ralphbean** [comment]
> Do you check if we need/have a behavior defined ONDELETE?

No, we should be fine.  Some filters might be left with zero rules attached, but that's okay.  They just won't trigger anything for anyone anymore.  The likelihood of someone having a filter defined with *only* one of these rules is also very low.

**Pull request merged**